### ISSUE TITLE
1.5.0beta1 not starting on FreeBSD

### ISSUE THREAD

**Issue by pascalj** [elasticsearch/logstash#2338]
Logstash does not start on FreeBSD. I tried setting `GEM_HOME` without success. Here's what's happening:

```
[web ~/logstash]$ bin/logstash
Errno::ENOENT: No such file or directory - /home/web/logstash/vendor/bundle/jruby/1.9/gems/activesupport-4.1.8/lib/i18n
                      file? at org/jruby/RubyFileTest.java:131
  contains_requirable_file? at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/basic_specification.rb:46
                       any? at org/jruby/RubyEnumerable.java:1473
  contains_requirable_file? at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/basic_specification.rb:46
                       any? at org/jruby/RubyEnumerable.java:1473
  contains_requirable_file? at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/basic_specification.rb:44
      find_inactive_by_path at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/specification.rb:898
                       find at org/jruby/RubyEnumerable.java:592
      find_inactive_by_path at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/specification.rb:897
               try_activate at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems.rb:183
                    require at /usr/home/web/logstash/vendor/jruby/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:132
               load_locale! at /home/web/logstash/lib/logstash/environment.rb:111
                     (root) at /home/web/logstash/lib/logstash/runner.rb:10
```

```
web:/root # uname -a
FreeBSD web 10.0-RELEASE-p12 FreeBSD 10.0-RELEASE-p12 #0: Tue Nov  4 05:07:17 UTC 2014     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64
```

```
web:/root # java -version
openjdk version "1.7.0_71"
OpenJDK Runtime Environment (build 1.7.0_71-b14)
OpenJDK 64-Bit Server VM (build 24.71-b01, mixed mode)
```

1.4.2 works perfectly on the same machine. Currently I have no idea why it looks for i18n at that place.  
If you need more info, just let me know.

**Comment by pascalj** [comment]
While trying to look into the issue, I built logstash from the 1-5-branch and I could not reproduce the bug anymore. The issue is fixed with 0d3df38cb9b — thanks!

**Comment by jordansissel** [comment]
A bit confused. 1.5.0 beta1 was released about 6 months after 0d3dd38 was merged - are you certain this is fixed?

**Comment by jordansissel** [comment]
Also, I'm not sure where you got instructions to set GEM_HOME, but bin/logstash overrides this setting and you should not attempt to set it.

**Comment by pascalj** [comment]
Sorry, I copied the wrong sha after bisecting. It was resolved with 516cc36e1a6f5 and I didn't investigate further. As for GEM_HOME: I manually set it in the bin/logstash script.
